packages/backend/src/models/ProjectModel/ProjectModel.ts

```diff
@@ -1102,7 +1103,17 @@ export class ProjectModel {
                     ? await trx('spaces')
                           .insert(
                               dbSpaces.map((d) => {
-                                  const createSpace = {
+                                  type CloneSpace = Omit<
```

Comment:
added some types to ensure all mandatory fields are present, remove `autogenerated` fields from the type.